src/main/java/mops/gruppen2/domain/model/group/Group.java

```diff
@@ -310,4 +315,8 @@ public String toString() {
                + (meta == null ? "meta: null" : meta.toString())
                + ")";
     }
+
+    public static Group EMPTY() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'EMPTY' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5366039)